examples/hotplugtest.c

```diff
@@ -22,12 +22,18 @@
 #include <stdio.h>
 
 #include "libusb.h"
+#include "libusbi.h"
 
 int done = 0;
 libusb_device_handle *handle = NULL;
 
 static int LIBUSB_CALL hotplug_callback(libusb_context *ctx, libusb_device *dev, libusb_hotplug_event event, void *user_data)
 {
+	UNUSED(ctx);
```

Comment:
This is also causing a build failure for earlier versions of Visual Studio. Variable declarations must come before any code in the current block scope.